scripts/eqtl_hail_batch/launch_generate_eqtl_spearman.py

```diff
@@ -129,6 +129,8 @@ def file_exists(files):
                 analysis_runner_output_path = output_prefix[5:].partition('/')[-1]
                 # get access level from bucket, rather than manual input
                 access_level = bucket_name.split('-')[-1]
+                if access_level == 'main':
+                    access_level = 'standard'
```

Comment:
It's cleaner if you don't derive this from the bucket name, but instead use the config value, so you don't rely on a particular bucket naming scheme:

```python
from cpg_utils.hail_batch import get_config
...
access_level = get_config()['workflow']['access_level']
```